codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/StructureGenerator.kt

```diff
@@ -285,6 +279,20 @@ class StructureGenerator(
                         .write("this.#L = #Q.invoke(block)", memberName, memberSymbol)
                         .closeBlock("}")
                 }
+
+                write("")
+
+                // render client side error correction function to set @required members
+                withBlock(
+                    "internal fun correctErrors(): Builder {",
```

Comment:
Should this be called something less negative like `fillMissingMembers()`?

Reply:
I named it to be close to the [specification](https://smithy.io/2.0/spec/aggregate-types.html#client-error-correction) that it comes from. Open to suggestions